packages/x-adapter-platform/src/types/responses/experience-controls-response.model.ts

```diff
@@ -0,0 +1,11 @@
+import { Dictionary } from '@empathyco/x-utils';
+
+/**
+ * Response for the `experience controls` endpoint.
+ *
+ * @public
+ */
+export interface PlatformExperienceControlsResponse {
+  controls: Dictionary;
+  events: Dictionary;
+}
```

Comment:
The response type in this package represents the structure of the endpoint's response. The endpoint is not returning neither `controls` or `events` properties, is returning a dictionary with anything inside, so this should be a `Dictionary<unknown>`.

```suggestion
export interface PlatformExperienceControlsResponse extends Dictionary<unknown> {}
```